release_notes/ocp-4-3-release-notes.adoc

```diff
@@ -133,6 +133,24 @@ changes.
 The pipelines build strategy is now depreciated. Use OpenShift Pipelines
 instead.
 
+[id="ocp-4-3-service-catalog-deprecated"]
+==== Deprecation of Service Catalog, Template Service Broker, Ansible Service Broker, and their Operators
+
+Service Catalog, Template Service Broker, Ansible Service Broker, and their
+associated Operators were deprecated in {product-title} 4.2 and will be removed
+in a future {product-title} release. If they are enabled in 4.3, the web console
+now warns the user that these features are still enabled.
+
+The `ServiceCatalogAPIServerEnabled` and
+`ServiceCatalogControllerManagerEnabled` alerts can be viewed from the
+*Monitoring* -> *Alerts* page and have a *Warning* severity.
+
```

Comment:
Since  Template Service Broker and Ansible Service were deprecated, we also have 
`AnsibleServiceBrokerEnabled`  and `TemplateServiceBrokerEnabled` alerts can be viewed from the *Monitoring* -> *Alerts* page and have a *Warning* severity.